pkg/dist/download-util.d.ts

```diff
@@ -0,0 +1,7 @@
+/**
+ * Downloads and extracts the tool at the specified URL.
+ *
+ * @url The URL of the tool to be downloaded.
+ * @returns The path to the locally extracted tool.
+ */
+export declare function downloadAndExtractTool(url: string): Promise<string>;
```

Comment:
Shouldn't the library's index.d.ts contain all exported types for the lib? I'm not sure why each individual sub-component needs a template file in dist.